tests/lib/external/snapd-testing-tools/.github/labeler.yml

```diff
@@ -0,0 +1,6 @@
+# Add 'needs tests' label to either any change on remote, tools or utils dir
```

Comment:
nitpick: the comment says 'needs test' but the actual key says 'need test', a miss-spelling? :-)

Reply:
Nice, I'll update that but it will be introduced in the next import